src/styles/less/ParentNavTopBar.less

```diff
@@ -0,0 +1,12 @@
+.parent-nav-top-bar {
+    position: sticky;
+    top: 0;
+    z-index: 100;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 2% 2%;
+    margin-bottom: 2%;
+    border-bottom: 1px solid gray;
```

Comment:
great work! You can add less variables for color.

Reply:
Thanks, Mosharraf! I avoided using any LESS variables due to the lack of styling this NavBar needed.

Reply:
Is this menu present at every viewport?

Reply:
Every viewport, that is correct. The menu will be visible on all devices--no matter screen size.
Just following the UX team's specifications here.